src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -1,4 +1,5 @@
 const refreshDropdown      = require('@binary-com/binary-style').selectDropdown;
+const Validation           = require('../../common/form_validation');
```

Comment:
could you also sort this alphabetically? we actually already have form_validation on line 9, perhaps you can remove `.validEmailToken` from that line, remove this line, and call that function using `Validation.validEmailToken` throughout the code below instead @Khalidxx